### ISSUE TITLE
Hot-reload issue in wsl

### ISSUE THREAD

**Issue by vinith-kovan** [latitude-dev/latitude#406]
## Steps To Reproduce

Steps we can take to reproduce the issue you're seeing. Sample data is helpful.

### Environment

- Node version (`node -v`): 18.20.2
- npm version (`npm -v`): 10.5.0
- Latitude CLI version (`latitude --version`): 1.5.0
- Latitude project version (check `latitude.json` at the root of your project): 1.1.6
- OS: windows 11
- Browser: brave and chrome
- Database: postgres

## Expected Behavior

Whenever changes made on code it should automatically reload the UI and shows the difference.

## Actual Behaviour

Instead it is showing white screen, I will attach a recording FYA.

https://github.com/latitude-dev/latitude/assets/156108204/f8d2d17b-465c-4bc4-9edf-01c9f96b94a4

## Workarounds

Anything you can do in advance of this being fixed?

No

**Comment by geclos** [comment]
@vinith-kovan thanks for reporting. To help us debug this I have some extra questions: 

What changes to code are you doing in the video attached? Changing views or queries? does the app break when doing a harmless change, for example when you are just adding a piece of text to the html, does it break in this case?

**Comment by vinith-kovan** [comment]
@geclos  Yes any kind of changes in view, queries and adding a piece of text, its breaking

**Comment by geclos** [comment]
@vinith-kovan I've been trying to reproduce your issue in WSL but I can't, could you provide a repo with the reproducible bug?
